### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MaryamIriagdam**
I have changed this file already.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,3 +8,4 @@
 debug-ashena/debug-ashena is a ✨ special ✨ repository because its `README.md` (this file) appears on your GitHub profile.
 You can click the Preview link to take a look at your changes.
 --->
+maryam iriagdam
```
Comment (somayeh-mohamadiyan) [inline README.md:11]: somayeh mohamadiyan

**Review by somayeh-mohamadiyan — COMMENTED** [review]
somayeh mohamadeyan

**Comment by mina-fadaei** [comment]
I am Mina Fadaei

**Comment by debug-ashena** [comment]
> I am Mina Fadaei   (Correct)

File: README.md [hunk 2]
```diff
@@ -8,3 +8,4 @@
 debug-ashena/debug-ashena is a ✨ special ✨ repository because its `README.md` (this file) appears on your GitHub profile.
 You can click the Preview link to take a look at your changes.
 --->
```
Comment (haniyehemadi76) [inline README.md:10]: https://github.com/haniyehemadi76/Internet-Advertisements/pull/1/commits/7d4bc17b252b18f2ec15a221c1f33ecac89fd8dd#diff-532b8509f8280980de02e7548afd7da8ec863399205139e25e2e9df620767a6c

**Comment by haniyehemadi76** [comment]
uploaded file

**Pull request closed without merging**